ruby-challenges.rb

```diff
@@ -10,23 +10,95 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def includes_letter(array, letter)
+    new_arr = []
+    array.each do |value|
+        if value.include?(letter) == true
```

Comment:
Consider how this solution could be simplified.  Hint: most of what you need for this method exists on these two lines.